packs/data/pathfinder-bestiary-3.db/vishkanya-infiltrator.json

```diff
@@ -721,12 +724,22 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>The vishkanya infiltrator is adept at dealing with tight situations. They have a +1 circumstance bonus to checks to Escape.</p>"
+                    "value": "<p>The vishkanya infiltrator is adept at dealing with tight situations. They have a +1 circumstance bonus to checks to @UUID[Compendium.pf2e.actionspf2e.Escape]{Escape}.</p>"
                 },
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "key": "FlatModifier",
+                        "predicate": [
+                            "action:escape"
+                        ],
+                        "selector": "skill-check",
```

Comment:
It should be athletics, acrobatics, and unarmed strikes.

Reply:
Can you tell me how to do the "unarmed-strike" thing? Just add "unarmed-strike" as an selector won't work for an NPC I guess?

Reply:
That can be skipped on this one anyways since they don't have an unarmed strike.